### DISCUSSION TITLE
[Bug] Incorrect color assignments compared to written legend for code frequency

### DISCUSSION THREAD

**Discussion by Deadlyelder** [community/community#8089] [Other Feature Feedback, Questions, & Ideas]
Within the code frequency of a repository, it states (as text) that addition and deletion to be blue and orange colored, respectively.
Yet, the group still retains the old green for addition and orange for deletion.

Attaching screenshot for reference.

<img width="956" alt="Screen Shot 2021-11-25 at 1 45 36 AM" src="https://user-images.githubusercontent.com/6140893/143331970-bf59bcfa-5f0f-45d3-b97b-89de20ed4b6e.png">

**Comment by MathiasReker** [comment]
Hello. You can report the bug here: https://support.github.com/contact/bug-report
